fungi/archives.md

```diff
@@ -1,7 +1,9 @@
-## Archive sites
+##Archive sites
```

Comment:
The space between the hashes and the text is actually a requirement in markdown headings. You can check using live preview in a markdown playground on StackEdit (https://stackedit.io/app) — if you remove the space, the heading gets interpreted as a simple paragraph.